assets/js/components/indicators.js

```diff
@@ -201,12 +201,15 @@ export class IndicatorSection {
       const response = await $.ajax({
         url: '/api/geography/geography/'
       });
-      const miifGrouped  = _.groupBy(response.results, "miif_category");
-      let similarGroup = miifGrouped[this.geography.miif_category];
 
-      // Remove current muni from selection
+      var filteredResults = response.results.filter(function( obj ) {
```

Comment:
a better name might be `activeMunicipalities` rather than `filteredResults` - it tells us more about what the variable represents